locale/English_en.ftl

```diff
@@ -923,7 +923,7 @@ generate_ids_instructions = It's simple, write the initial id in the box below a
 generate_ids_accept = Accept
 
 context_menu_delete_filtered_out_rows = Delete Filtered-out Rows
-are_you_sure_delete_filtered_out_rows = This will delete all filtered-out rows. Are you sure?
+are_you_sure_delete_filtered_out_rows = Are you sure to delete all filtered-out rows?
```

Comment:
Same with this one. It clearly specifies what's going to happen. Any reason for the rephrasing?

Reply:
Same here. This text has two references. In the first sentence, "This" refers to this action, but what is this action? Users must recall hardly. #247 demonstrates this problem too. From a static screenshot, how do you "this" refer to what? You must use your experience and match the text in dialog with all text in this program, and then tell what situations will give this text. The text itself doesn't tell.

The second reference is implicit in "are you sure".

It's not a minor issue. The references increase user's cognition load because the reader has to parse the two references. Why don't we simply say "Are you sure to delete all filtered-out rows?" It tells what will happen, and it contains no pronoun.  Readers don't have to figure out the object a pronoun refers to.

Reply:
Same as with the other one. Ok, but put the "you want to" after "are you sure".